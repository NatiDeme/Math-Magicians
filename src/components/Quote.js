import React from 'react';
import '../style/Nav.css';

const NavBar = () => (
  <div className="quote-page">
    <p>
      Math is beautiful. But sometimes, this can be hard to see,
      and even harder to convey to students who don’t — yet — share your passion.
    </p>
  </div>
);

export default NavBar;
