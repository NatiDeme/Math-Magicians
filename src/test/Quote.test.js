import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote'


it('renders correctly', () => {
  const docment = renderer
  .create(<Quote />).toJSON();
  expect(docment).toMatchSnapshot();
});