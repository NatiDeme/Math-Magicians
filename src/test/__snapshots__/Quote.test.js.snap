// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="quote-page"
>
  <p>
    Math is beautiful. But sometimes, this can be hard to see, and even harder to convey to students who don’t — yet — share your passion.
  </p>
</div>
`;
